docker-compose-integration.yaml

```diff
@@ -11,23 +11,6 @@ services:
     links:
       - jellyfish
```

Comment:
links are deprecated, you should create a network https://www.baeldung.com/ops/docker-compose-links-depends-on#docker-compose-network